bubblewrap.c

```diff
@@ -1974,6 +1993,15 @@ main (int    argc,
     {
       /* Parent, outside sandbox, privileged (initially) */
 
+      if (opt_info_fd != -1)
+        {
+          cleanup_free char *output = xasprintf ("{\n    \"child-pid\": %i\n}\n", pid);
+          size_t len = strlen (output);
+          if (write (opt_info_fd, output, len) != len)
+            die_with_error ("Write to info_fd");
+          close (opt_info_fd);
+        }
+
```

Comment:
I need this so that the caller knows the child-pid before it can set up the user namespace.  AFAICS it should be safe to use write (EPERM is raised only when a file seal is used) but I'll rearrange the order of these operations so we write only after `drop_privs()`